lib/resources/example_json/r4_examples_allergy_intolerance.rb

```diff
@@ -84,6 +84,329 @@ module Resources
       ]
     }.freeze
 
+    R4_ALLERGY_INTOLERANCE_REVINCLUDE_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '65ee4c79-3514-4eab-b606-10ad56fb0d5d',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/AllergyIntolerance?_id=74771957%2CS-75479691&_revinclude=Provenance%3Atarget'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/AllergyIntolerance/74771957',
```

Comment:
Update tenant to https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d

Reply:
changes addressed